packages/developer-console-spa/src/components/ConfigureLighthouse.tsx

```diff
@@ -0,0 +1,237 @@
+  const openOPLighthouseDoc = () => {
+    window.open('/get-started/docs/apps/internal/lighthouse', '_blank',);
+  };
+
+  const onCardExpand = () => {
+    setIsCardExpanded( !isCardExpanded );
+  };
+
+  return (
+    <>
+      <Stack hasGutter>
+        <StackItem>
+          <Header title="Configure Lighthouse" />
+        </StackItem>
+        { !isProjectLinked ?
+          <StackItem>
+            <Card isRounded>
+              <CardBody>
+                <EmptyState>
+                  <EmptyStateBody>
+                    <img src={ lhIllustration } alt="Lighthouse Illustration" />
+                  </EmptyStateBody>
+                  <Button onClick={ handleModalToggle } variant="primary">Get Started</Button>
+                  <EmptyStateSecondaryActions>
+                    <Button variant="link" onClick={ openOPLighthouseDoc }>Learn more</Button>
+                  </EmptyStateSecondaryActions>
+                </EmptyState>
+              </CardBody>
+            </Card>
+          </StackItem>
+        :
+          <StackItem>
+            <Card>
+              <CardHeader>
+                <CardActions>
+                  <Button
+                    aria-label="Delete Branch"
+                    alt="Delete Branch" variant="link"
+                  >
+                    <TrashIcon />
+                  </Button>
+                </CardActions>
+                <CardTitle>
+                  <CheckCircleIcon /> You are all set
+                </CardTitle>
+              </CardHeader>
+              <CardBody>
+                <Flex>
+                  <FlexItem align={ { default: 'alignRight' } }>
+                    <Button variant="link" onClick={ openLighthouseDoc }>Learn More</Button>
+                  </FlexItem>
+                  <FlexItem>
+                    <Button variant="link">View my Lighthouse reports    <ExternalLinkAltIcon /></Button>
+                  </FlexItem>
+                </Flex>
+              </CardBody>
+            </Card>
+            <br />
+            <Card id="card2" isExpanded={ isCardExpanded }>
+              <CardHeader
+                onExpand={ onCardExpand }
+                toggleButtonProps={ {
+                  id: 'toggle-button',
+                  'aria-label': 'Details',
+                  'aria-labelledby': 'titleId toggle-button',
+                  'aria-expanded': isCardExpanded
+                } }
+              >
+                <CardTitle>
+                  How do you use lighthouse CI?
+                </CardTitle>
+              </CardHeader>
+              <CardExpandableContent>
+                <CardBody>
+                  <p>To get started using Lighthouse, just add the following code into 'lighthouserc.js' file to your project root:</p>
+                  <ClipboardCopy isExpanded={ true } isCode isReadOnly hoverTip="Copy" clickTip="Copied" variant={ ClipboardCopyVariant.expansion }>
+                    {
+                      `module.exports = {
```

Comment:
Here as its a templating string the left spacing has an effect.

 Can we remove some of that space so that in the UI this part doesn't feel too spaced out like now.

Reply:
Done